config/csi-rbac/cephfs_ctrlplugin_cluster_role.yaml

```diff
@@ -63,3 +63,6 @@ rules:
   - apiGroups: [""]
     resources: ["serviceaccounts/token"]
     verbs: ["create"]
+  - apiGroups: ["authentication.k8s.io"]
```

Comment:
i thought we need to add only to the Roles not to the clusterRole, can you please add it to Role and verify csi-addons functionality works?